chpl/chpl-service/src/main/java/gov/healthit/chpl/validation/surveillance/reviewer/NewSurveillanceRemovedCriteriaReviewer.java

```diff
@@ -64,6 +66,18 @@ private void checkRequirementForRemovedCriteria(Surveillance surv, SurveillanceR
         }
     }
 
+    private void checkRequirementForRemovedTransparency(Surveillance surv, SurveillanceRequirement req) {
+        if (req.getType() != null && !StringUtils.isEmpty(req.getType().getName())
+                && req.getType().getName().equalsIgnoreCase(SurveillanceRequirementType.TRANS_DISCLOSURE_REQ)) {
+            String requirement = req.getRequirement();
+            if (requirement != null && requirement.equalsIgnoreCase("170.523 (k)(2)")) {
+                surv.getErrorMessages().add(
+                        msgUtil.getMessage("surveillance.requirementNotAddedForRemovedRequirement",
```

Comment:
Is this error message only supposed to be added for ACBs? I could definitely be missing some larger context here but it looks like it might give an error for all users.

Reply:
This method is called from the `review` method, but only after checks for both a flag and the user being ROLE_ACB. Yes, only ROLE_ACB will get this message; ROLE_ADMIN and ROLE_ONC never get into this method at all